Split section for guide on Relays and App Views (#518)
bluesky-social/atproto-website: src/app/[locale]/guides/self-hosting/en.mdx

- [Tranquil PDS](https://tangled.org/lewis.moe/bspds-sandbox), written in Rust.
- [Cocoon](https://tangled.org/hailey.at/cocoon), written in Go.

## Relays

> Hosting your own Relay is possible, though bandwidth intensive. Relays typically operate in a full-network mode, where they replicate data from all PDSes and repositories on the AT Protocol network. However, operating a Relay is *not* storage-intensive, as Relays are not required to retain archival copies of network data as of Sync 1.1.

Author: ```suggestion
Hosting your own Relay is possible, though bandwidth intensive. Relays typically operate in a [full-network mode](/guides/the-at-stack#relays), where they replicate data from all PDSes and repositories on the AT Protocol network. However, operating a Relay is *not* storage-intensive, as Relays are not required to retain archival copies of network data as of Sync 1.1.
```

> Hosting your own Relay is possible, though bandwidth intensive. Relays typically operate in a [full-network mode](/guides/the-at-stack#relays), where they replicate data from all PDSes and repositories on the AT Protocol network. However, operating a Relay is *not* storage-intensive, as Relays are not required to retain archival copies of network data as of Sync 1.1.

Author: ```suggestion
Hosting your own Relay is possible, though bandwidth intensive. Relays typically operate in a [full-network mode](/guides/the-at-stack#relays), where they replicate data from all PDSes and repositories on the AT Protocol network. However, operating a Relay is *not* storage-intensive, as Relays are not required to retain archival copies of network data as of [Sync 1.1](/blog/relay-updates-sync-v1-1).
```

@axfelix can this be linked like this? or does it need the full url? `https://atproto.com/blog/relay-updates-sync-v1-1`

> Hosting your own Relay is possible, though bandwidth intensive, as generally relays operated in full-network mode, where they replicate data from all PDSes and repositories on the AT Protocol network. Previously operating a relay was also resource intensive due to data storage requirements, as of Sync 1.1, the data storage requirements are no longer an issue, as relays are not required to be archival.

Reviewer: I may wordsmith this a bit so as not to lead with "previously," eg:

```suggestion
Hosting your own Relay is possible, though bandwidth intensive. Relays typically operate in a full-network mode, where they replicate data from all PDSes and repositories on the AT Protocol network. However, operating a Relay is *not* storage-intensive, as Relays are not required to retain archival copies of network data as of Sync 1.1.
```

> Hosting your own Relay is possible, though bandwidth intensive, as generally relays operated in full-network mode, where they replicate data from all PDSes and repositories on the AT Protocol network. Previously operating a relay was also resource intensive due to data storage requirements, as of Sync 1.1, the data storage requirements are no longer an issue, as relays are not required to be archival.

Reviewer: Blog links are a-OK imo (they're on the same domain now, even!)

> Hosting your own Relay is possible, though bandwidth intensive, as generally relays operated in full-network mode, where they replicate data from all PDSes and repositories on the AT Protocol network. Previously operating a relay was also resource intensive due to data storage requirements, as of Sync 1.1, the data storage requirements are no longer an issue, as relays are not required to be archival.

Reviewer: Probably just link the words [full network] to (/guides/the-at-stack#relays)

There are a few different relay implementations:
- [Reference Implementation](https://github.com/bluesky-social/indigo/blob/main/cmd/relay/README.md), written in Go.
- [Blacksky Relay](https://github.com/blacksky-algorithms/rsky/tree/main/rsky-relay#rsky-relay), written in Rust.

## Tap

Tap is used for syncing existing records from the AT network and then continuing to receive new records from the firehose. For a guide to deploying Tap, refer to the [Tap readme](https://github.com/bluesky-social/indigo/blob/main/cmd/tap/README.md#distribution--deployment).

We also provide a quick start guide for [deploying Tap](https://github.com/bluesky-social/indigo/blob/main/cmd/tap/RAILWAY_DEPLOY.md) to [Railway](https://railway.com/).

## AppViews

> Creating or hosting your own AppView is possible, but resource intensive. Compared with hosting your own PDS, which is a way of personally controlling access to your own data repository, hosting an AppView is something that you would only do to provide application-level infrastructure. Generally, the first thing you would need to do here is replicate all of the data in the network for the Lexicons that your application handles. Refer to the [Sync](/guides/sync) docs for a starting point.

Author: @axfelix I think most application developers will probably want to go the Tap route, rather than implementing Sync themselves directly. The learning curve is significantly lower for Tap.

Maybe we say: "Refer to the Tap section above, or to the [Sync] docs for more information" ?

## Deploy Recipes

> For community-contributed deployment solutions that do not use our official tooling, and may support different environments, refer to the [deploy-recipes](https://github.com/bluesky-social/deploy-recipes) Github repo. Contributions are welcome!

Reviewer: Yup, agree this is sloppy wording, probably should have gotten revised from an earlier draft!
